sql/analyzer/analyzer.go

```diff
@@ -387,6 +387,7 @@ func prePrepareRuleSelector(id RuleId) bool {
 	case
 		// OnceBeforeDefault
 		reresolveTablesId,
+		validatePriviledgesId,
```

Comment:
we're skipping privilege checks in pre and post?

Reply:
Meant to just skip in pre, will add it back in post.
I did this because the analyzer would call `CheckPrivileges` on an unresolved TableFunction in pre, and give panics.